### ISSUE TITLE
Keybinds issues

### ISSUE THREAD

**Issue by joex92** [TeamAOF/All-of-Fabric-6#303]
the modpack is so big and complete that a lot of mods already use same keybinds... so I suggest for the modpack to have all keybinds preset in a way that there are no conflicts between any of the mods, it's so hard sometimes to understand how to control some of the mods or how to use them, specially because of the keybinds, for example, you can't open the Traveler's Backpack if you equip it while having equipped an inmi backpack, because both use the letter `B` to open the backpack... 

So I started looking for this topic and after reading #63, I found this mod: https://github.com/Klotzi111/amecs-api
which only adds the possibility to use modifiers on keybinds, Already tried and no conflicts whatsoever... 

So I recommend to add this mod and/or preset all keybinds from all the mods so there are no conflicts...

**Comment by Linguardium** [comment]
it is impossible to preset all the keybinds to remove conflicts and fabric does not have a current key processing event that allows passing handling to another mod (unfortunately it went stale and was never implemented)

iirc, the last time amecs was looked at, it was missing several search capabiliities but that was a long time ago so they may have updated to be more feature complete

**Comment by joex92** [comment]
no, I'm not suggesting Amecs but Amecs-API... Amecs has conflicts with the search capability, because of some things that Amecs adds, but Amecs-API doesn't add those things, just adds the possibility of using modifiers on the control menu: 
![image](https://github.com/TeamAOF/All-of-Fabric-6/assets/645579/e3ada1d4-336d-4e9b-b448-c6d8561c97f6)

**Comment by joex92** [comment]
yes, it works fine, the only "issue" (maybe the dev didn't think about this?) with Amecs-API is that apparently it doesn't recognize mouse extra buttons, so `Button 4` and up won't work... 🤷🏼‍♂️ I'm going to report this in their github... 
I don't think this is a big issue, I don't think the majority of users have a mouse with extra buttons 🤔

**Comment by WraithStar** [comment]
basically a dupe of #79
